.ci/gcb-generate-diffs.yml

```diff
@@ -1,4 +1,7 @@
 ---
+concurrency:
```

Comment:
I am having trouble finding documentation for `concurrency` in cloudbuild yaml files - could you provide a link?